Tests/Realm.Tests/Database/NotificationTests.cs

```diff
@@ -102,6 +102,24 @@ public void ListShouldSendNotifications()
             }
         }
 
+        [Test]
+        public void ShouldNotSendNotifications()
+        {
+            var person = _realm.Write(() => _realm.Add(new Person()));
+            var notificationCount = 0;
+            person.PropertyChanged += (sender, e) =>
+            {
+                notificationCount++;
+            };
+
+            _realm.Write(() =>
+            {
+                person.Friends.Add(new Person());
+                _realm.Refresh();
+                Assert.That(notificationCount, Is.EqualTo(0));
+            });
+        }
+
```

Comment:
You need to call refresh after the write, otherwise there is nothing to refresh here. 
Another test would be to check if you're actually receiving notifications for all the properties of Person

Reply:
Other tests would be also to modify one of the linked persons and see that the notification does not get raised (it's already like this actually)

Reply:
Also, definitely take a look at the CI results